gradle/distributionTesting.gradle.kts

```diff
@@ -44,8 +44,8 @@ tasks.withType<DistributionTest> {
         systemProperties[integTestVersionsSysProp] = "latest"
     }
 
-    fun ifProperty(name: String, then: String): String? =
-        then.takeIf { hasProperty(name) && property(name).toString().isNotEmpty() }
+    fun Project.ifProperty(name: String, then: String): String? =
+        then.takeIf { findProperty(name) != null }
```

Comment:
Isn't that then simply `hasProperty(name)`?

Reply:
We happen to only set the property to `true`.
It should be `findProperty(name) == true`.

Reply:
Fixed in bb566c460eb4c77296f7e57633b65e55ddd0913f